src/General/Alert/Alert.tsx

```diff
@@ -140,7 +126,7 @@ export interface Props {
   /** A function to close Alert. onClose will be triggered if the close icon is clicked or the Alert has ran its autoClose duration or ESC/Enter is pressed */
   onClose(): void;
```

Comment:
Since `onClose` is required here, I guess probably we don't have an uncontrolled Alert. We can create an issue to investigate it later.